app/src/main/java/org/dhis2/usescases/enrollment/EnrollmentActivity.kt

```diff
@@ -255,6 +257,7 @@ class EnrollmentActivity : ActivityGlobalAbstract(), EnrollmentView {
 
     @Deprecated("Deprecated in Java")
     override fun onBackPressed() {
+        super.onBackPressed()
```

Comment:
why do we need calling the super?

Reply:
I was getting this warning that "Overriding method should call super.onBackPressed" so that is why I added the call super. But I have removed it anyway.